en/views/cells.rst

```diff
@@ -276,3 +276,28 @@ This will allow us to define the option when creating the cell::
 Cell options are handy when you want data available as properties allowing you
 to override default values.
 
+Using Helpers inside a Cell
+===========================
+
+Since cells have their own View scope they don't share the helpers defined in
+`src/View/AppView.php`. You need to load those helpers again inside your cell::
```

Comment:
This is incorrect.  Cells have their own context and their own View instance but it's still an AppView instance and helpers loaded in AppView::initialize() would be available.

Reply:
I now have found my root problem. I thought I had loaded the plugins helper inside my AppView initialize method but I didn't. Thats why I was confused why the Helper wasn't available inside the cell.

I will adjust my statement.